github/event_types.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
```

Comment:
maybe add a file-level comment that these event types are shared between the Events API and used as Webhook payloads.